soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -531,23 +536,23 @@ void CosmeticsUpdateTick() {
 void ApplyOrResetCustomGfxPatches(bool manualChange) {
     static CosmeticOption& magicFaroresPrimary = cosmeticOptions.at("Magic.FaroresPrimary");
     if (manualChange || CVarGetInteger(magicFaroresPrimary.rainbowCvar, 0)) {
-        static Color_RGBA8 defaultColor = {magicFaroresPrimary.defaultColor.x, magicFaroresPrimary.defaultColor.y, magicFaroresPrimary.defaultColor.z, magicFaroresPrimary.defaultColor.w};
+        static Color_RGBA8 defaultColor = {magicFaroresPrimary.defaultColor.r, magicFaroresPrimary.defaultColor.g, magicFaroresPrimary.defaultColor.b, magicFaroresPrimary.defaultColor.a};
         Color_RGBA8 color = CVarGetColor(magicFaroresPrimary.cvar, defaultColor);
```

Comment:
Now that the default colors are `Color_RGBA8` there's no need to do the conversion, same with the other instances of this.
```suggestion
        Color_RGBA8 color = CVarGetColor(magicFaroresPrimary.cvar, magicFaroresPrimary.defaultColor);
```

Reply:
Great point, taken care of.